### PR TITLE
Enforce priority label

### PR DESCRIPTION
**Pull request by JamesAlfonse**
Added yml file into the workflows folder.

### CODE REVIEW

**Comment by JamesAlfonse** [comment]
The automation that enforces priorities will only work if the label already exists. The automation that creates the labels runs every day at midnight (as long as it has the permissions in the repositories to do so).

File: workflows/Create_Labels.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: Ensure Standard Labels Across Repositories
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'  # Runs daily at midnight
```
Comment (JFWooten4) [inline workflows/Create_Labels.yml:6]: Love the clear comment here! 💜

File: workflows/Create_Labels.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+jobs:
+  create-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      administration: write
```
Comment (JFWooten4) [inline workflows/Create_Labels.yml:13]: `write` permissions needed and applied for label management across repositories. 👍🏼

File: workflows/Create_Labels.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      - name: Create Labels
+        run: node create-labels.js
+
+      - name: Install dependencies
+        run: npm install @octokit/core
+
+      - name: Create Labels Script
+        run: |
+          const { Octokit } = require("@octokit/core");
+          const octokit = new Octokit({ auth: `token ${process.env.GITHUB_TOKEN}` });
```
Comment (JFWooten4) [inline workflows/Create_Labels.yml:32]: Just want to make sure we've thought about the security policies around the token, as this is something I'm not familiar with. 🔐 Ideally, we might incorporate it into a global [communication policy](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) here 💭

**Review by JFWooten4 — COMMENTED** [review]
Stellar work getting this configured, ensuring very readable code, and ideating an ingenious plan to synchronize it across all the repos! 🚀

**Pull request merged**